src/main/resources/assets/tconstruct/book/puny_smelting/en_us/upgrades/ranged/piercing.json

```diff
@@ -0,0 +1,14 @@
+{
+  "modifier_id": "tconstruct:piercing",
+  "text": [
+    {
+      "text": "Tool gains the power of the cactus, attacking through armor."
```

Comment:
May want a blurb here, "Not to be confused with impaling" as vanilla calls impaling piercing